### DISCUSSION TITLE
[RESOLVED] How to run xgen?

### DISCUSSION THREAD

**Discussion by phinguyen202** [xuri/xgen#29] [Q&A]
Hi there,

I'm a newbie for Go. So sorry for the very basic question.
How do to run xgen?

I try many time but always got the following:
 `'xgen' is not recognized as an internal or external command,
operable program or batch file.`

I have set up project as following:
![image](https://user-images.githubusercontent.com/23725530/130277466-e0d02d86-a6bb-4cb4-b972-65fdbecbf06c.png)

Many thanks!
Phi
-------------
RESOLVED
Solution: Run file xgen.go directly

**Comment by Eusebiotrigo** [comment]
Better if you configure your GOPATH

https://go.dev/doc/gopath_code#GOPATH

export PATH=$PATH:$(go env GOPATH)/bin
